### ISSUE TITLE
Make UserTable tests pass

### ISSUE THREAD

**Issue by nucleogenesis** [learningequality/kolibri#12664]
## Observed behavior

See https://github.com/learningequality/kolibri/pull/12662 - you will need to change the tests which are ignored there using the `xit()` function to use `it` and test the same cases.

I did not find it easy to work out how to fix the tests without rewriting them. The way I understand it, the behavior being tested is actually behavior that is defined in a parent component. 

That is to say, `UserTable` is setup to accommodate the `v-model` - so it's own `this.value` prop indicates what ids are selected. Then, when a checkbox is clicked, we emit the `input` event with a payload of what `this.value` should be after the checkbox was clicked. 

The parent then will update whichever of its own local values it is passing to the `UserTable` `v-model` and can use it elsewhere.

These tests directly mount `UserTable` - so I think that the issue is to do with the fact that there is no parent component mounted w/ UserTable that is reacting to the `v-model` relationship. So, in these tests, `UserTable` is emitting events into the aether.

## Expected behavior

Tests in UserTable spec should not be ignored.

## User-facing consequences

n/a

## Context

`develop`

**Comment by Suchitra-Sahoo** [comment]
@nucleogenesis I would like to work in it. Kindly assign this issue to me.

**Comment by AlexVelezLl** [comment]
Hey @Suchitra-Sahoo! Thanks for your interest in contributing to this issue. I will assign this to you. If you have any question, please let us know! 👐

**Comment by MisRob** [comment]
Hi @Suchitra-Sahoo, are you still planning to work on this?

**Comment by Suchitra-Sahoo** [comment]
> Hi [@Suchitra-Sahoo](https://github.com/Suchitra-Sahoo), are you still planning to work on this?

Unfortunately, due to some unforeseen circumstances, I wasn't able to work on it as planned. Please feel free to unassign me so someone else can take it up.

**Comment by Abhishek-Punhani** [comment]
Hi @MisRob Can i work on this issue  ?

**Comment by MisRob** [comment]
Yes @Abhishek-Punhani, thank you. Please follow [this guide](https://kolibri-dev.readthedocs.io/en/develop/frontend_architecture/unit_testing.html). Note that not all test suites are yet implemented via VTL - if this one is not migrated yet, I'd recommend to switch to VTL on this opportunity since its principles will guide you and help you write meaningful and robust tests.

**Comment by Abhishek-Punhani** [comment]
Hi @MisRob  I have raised a PR , Please review it at your convenience.Thanks!

**Comment by nucleogenesis** [comment]
Closed by #13088

**Comment by MisRob** [comment]
Thanks a lot @Abhishek-Punhani
